README.md

```diff
@@ -41,6 +41,7 @@ Environment variables and their default values,
 * `POSTGRES_DB_USER=admin`
 * `POSTGRES_DB_PASSWORD=secret`
 * `POSTGRES_DB_QUERY_LIMIT=10`
+* `DOMAIN=assistant.bible`
```

Comment:
Good to have the new ENV variables `SUPABASE_URL` and `SUPABASE_KEY`(not with value) listed here

Reply:
Done